main.py

```diff
@@ -185,9 +186,13 @@ async def get_and_delete_old_versions(image_name: ImageName, inputs: Inputs, htt
             # Skipping because no tagged images should be deleted
             continue
 
-        if any(tag in inputs.skip_tags for tag in image_tags):
-            # Skipping because this image version is tagged with a protected tag
-            continue
+        skip = False
+        for skip_tag in inputs.skip_tags:
+            if any(fnmatch(tag, skip_tag) for tag in image_tags):
+                # Skipping because this image version is tagged with a protected tag
+                skip = True
+        if skip:
+            break
```

Comment:
Why break instead of continue?
```suggestion
            continue
```

Reply:
no reason except for I did it wrong :D

Reply:
fixed this and pushed